this is a bug fix 

this is file3.txt created locally
